docs/packages/markdown/prism.js

```diff
@@ -7,7 +8,218 @@ require('prismjs/components/prism-jsx');
 require('prismjs/components/prism-markup');
 require('prismjs/components/prism-tsx');
 
+function createPrismComponentLinkerHook(getComponentApiPageLink) {
+  return prism.hooks.add('wrap', (env) => {
+    if (env.type === 'class-name') {
+      const possiblyComponentIdentifier = env.content;
+      const componentApiPage = getComponentApiPageLink(possiblyComponentIdentifier);
+      if (componentApiPage) {
+        env.content = `<a href="/api/${componentApiPage}">${env.content}</a>`;
+      }
+    }
+  });
+}
+
+let componentLinkerHook = null;
+
+const apiPages = [
```

Comment:
I guess there is a challenge around keep this array up to date.

Reply:
> What about the developers that use mouse down to select one example in a demo? It seems to be a bit harder. Should we care? I could find two cases: Developers that start the selection a bit too soon are prevented to select anything by the <a>. Developers that are used to double click on a component name to copy it.

These fantastical people you constructed already have problems with selecting any text on the web. 

> Does it prevent the implementation of live editing? At least, in the shape, it was POC in [docs] Live editable demos #24640?

live editing is still not suitable for documentation. This is another reason why.